### PR TITLE
Docs: Fix small typo in tutorial docs

### PR DESCRIPTION
**Pull request by Orlandster**

### CODE REVIEW

File: doc/TUTORIAL.md [hunk 1]
```diff
@@ -148,7 +148,7 @@ Description of the code:
    - `pattern` defines the variable part of the route, everything that comes after `/example/`. It can include any
      number of named parameters. These are converted into
      regular expressions by [`path-to-regexp`][path-to-regexp]. the list of the valid categories can be seen
-6. `route()` declares the URL path at which the service operates. It also maps components of the URL path to handler parameters.
+6. `handle()` declares the URL path at which the service operates. It also maps components of the URL path to handler parameters.
```
Comment (paulmelnikow) [inline doc/TUTORIAL.md]: Should this item in the list be removed instead? It looks like items 5 and 6 are duplicates.

Comment (PyvesB) [reply doc/TUTORIAL.md]: Well, we still need a text description for number 6 in the code. Maybe replace 6 with 8, and merge 5 and 7 together?

Comment (paulmelnikow) [reply doc/TUTORIAL.md]: Ah, right, these numbers correspond with the code comments. Yea, what you're suggesting makes sense.

Comment (chris48s) [reply doc/TUTORIAL.md]: Yes the description for the `handle()` function should be:

> All badges must implement the `async handle()` function that receives parameters to render the badge...

and the description for `route()` should be

> `route()` declares the URL path at which the service operates. It also maps components of the URL path to handler parameters.

its the numbering that has become mixed up.

**Comment by paulmelnikow** [comment]
Hey @Orlandster would you like to update this?

**Comment by PyvesB** [comment]
I've included the fixes discussed here in  #3593. Closing.

**Pull request closed without merging**